roles/openshift_metrics/tasks/generate_certificates.yaml

```diff
@@ -126,7 +124,7 @@
     '{{ hawkular_metrics_password.stdout }}'
 - name: read files for the hawkular-metrics secret
   command: >
-    base64 --wrap 0 "{{ mktemp.stdout }}/certs/{{ file }}"
+    base64 --wrap 0 "{{ mktemp.stdout }}/certs/{{ item }}"
```

Comment:
@ewolintez has recommanded we migrate to using 'loop_variable' to override the use of 'item' in order to defend against item being populated somewhere else.